bin/build

```diff
@@ -2,7 +2,7 @@
 load File.expand_path(File.join(File.dirname(__FILE__), "..", "lib", "vanagon.rb"))
 
 optparse = Vanagon::OptParse.new("#{File.basename(__FILE__)} <project-name> <platform-name> [<target>] [options]",
-                                 [:workdir, :configdir, :engine, :preserve, :verbose])
+                                 [:workdir, :configdir, :engine, :preserve, :verbose, :skipcheck])
```

Comment:
Think it's possible to call it "`--skip-check`" instead of "`--skipcheck`"? Nonblocking nitpick obviously.